CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+This is an adapted version of the contributing guide used by the Community Committee: https://github.com/nodejs/community-committee/blob/master/CONTRIBUTING.md
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request,
+  or contributing in some other way.
+* A **Collaborator** is a contributor who has been given write access to the repository.
+
+# Contributions
+
+Any change to resources in this repository must be through pull requests. This applies to all changes
+to documentation, code, binary files, etc. 
+No pull request can be merged without being reviewed.
+
+The default for each contribution is that it is accepted once no collaborator has an objection.
+During review collaborators may also request that a specific contributor who is most versed in a
+particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off"
```

Comment:
Are we not using the github review tools? 🙃

Reply:
hmm, this integrates with the github review tools. not sure what you mean?

Reply:
Sorry, just being a little (very?) nitpicky. I read this as the PR has to be approved with an LGTM-comment, and using the review tools is not sufficient.

Reply:
oh, i'm fairly sure that the review tools are sufficient for giving a LGTM most of the time